src/styles/index.css

```diff
@@ -77,10 +77,19 @@ html .akEditor > div:first-child {
 
 /* prose mirror toolbar in the upper bar */
 html .akEditor > div:first-child {
-    margin-bottom: 0;
+    width: 100%;
     flex-wrap: wrap;
     height: auto;
-    box-shadow: 0 11px 9px 0 rgba(0, 0, 0, 0.08), 0 3px 3px 0 rgba(50, 54, 63, 0.08);
+    position: fixed;
+    top: auto;
+    bottom: 1rem;
```

Comment:
What about the `safe-area` for mobile browser? Should we respect it?

Reply:
I am sorry I am not sure what you're referring to in this context, can you please elaborate?

Reply:
On mobile we've a concept called "safe area" that refers to the space where... it's safe to display content. For exemple  iPhone X and + have a "gray bar" that you can drag to go back to the home screen. Since this "gray bar" is interactive it's not safe to display content close to it. (https://developer.apple.com/documentation/uikit/uiview/positioning_content_relative_to_the_safe_area) 

There are few CSS's vars to handle that correctly. You can take a look at our CozyDialog implementation in Cozy UI if you want. (for instance https://github.com/cozy/cozy-ui/pull/1677) 

Since this code seems to have a fixed position at bottom: 1rem I'm asking if we should not take care of this safe are

Reply:
I suggest  `bottom: calc(env(safe-area-inset-bottom, 1rem) + 1rem);`